pkg/backend/display/progress_test.go

```diff
@@ -120,3 +124,116 @@ func TestProgressEvents(t *testing.T) {
 		})
 	}
 }
+
+// The following test checks that the status display elements have retain on delete details added.
+func TestStatusDisplayFlags(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
```

Comment:
optional nit: The list of test cases in a table test is typically just called "tests", and the individual test cases "tt".